Lib/warnings.py

```diff
@@ -77,6 +83,9 @@ def _formatwarnmsg_impl(msg):
                 if line:
                     line = line.strip()
                     s += '    %s\n' % line
+        elif not tracing:
+            s += (f'{category}: Enable tracemalloc to get the object '
+                  f'allocation traceback\n')
```

Comment:
Question: Would be useful to add to this sentence how to enable tracemalloc? For example:

```python
s += (f'{category}: Enable tracemalloc to get the object '
          f'allocation traceback (i.e. -X tracemalloc=N_FRAMES)\n')
```